src/components/NewBallot.jsx

```diff
@@ -242,26 +244,50 @@ export class NewBallot extends React.Component {
         .utc()
         .add(constants.startTimeOffsetInMinutes, 'minutes')
         .unix()
-      methodToCreateBallot(startTime)
-        .on('receipt', async tx => {
-          commonStore.hideLoading()
-          if (tx.status === true || tx.status === '0x1') {
-            const newId = Number(tx.events.BallotCreated.returnValues.id)
-            const card = await contractsStore.getCard(newId, contractType)
-            ballotsStore.ballotCards.push(card)
 
-            swal('Congratulations!', messages.BALLOT_CREATED_SUCCESS_MSG, 'success').then(result => {
-              push(`${commonStore.rootPath}`)
-              window.scrollTo(0, 0)
-            })
+      web3.eth.sendTransaction(
```

Comment:
`web3.eth.sendTransaction` is defined 3 times with a similar code. Let's define the function in `src/helpers.js` and reuse it in components.

Reply:
@vbaranov ok, I've done it in the last two commits: https://github.com/poanetwork/poa-dapps-voting/pull/158/commits/7fa26a17f1a5c29f7d63ee71e1f24858df4f99a9 and https://github.com/poanetwork/poa-dapps-voting/pull/158/commits/99a712f7d75e07553f66106f989edc3b80f5bf07. DApp was tested with `poa-test-setup` after these changes.